src/components/app-layout/components/skip-link/index.marko

```diff
@@ -0,0 +1,37 @@
+<a#skip-link href=`#${input.anchor}`>
+  Skip to main content
+</a>
+
+style {
+  /* This positions the skip link off-page accessibly, and transitions it in noticeably when it receives focus.
+     More info: https://webaim.org/techniques/skipnav/
+  */
+   #skip-link { /* Intentionally using an id for specificity reasons */
+    position: fixed;
+    z-index: 10000000;
+    left: 0;
+    top: -4em; /* Fallback for browsers without `transform` support */
+    padding: 0.2em;
+    background: #fff;
+    border: 2px solid hsl(0, 0%, 25%);
+    border-bottom-right-radius: 0.25em;
+    text-decoration: underline; /* Not a fan of how we remove this for all links by default */
+    transform: translateY(-100%);
+    transition: top 1s steps(1), transform 1s; /* Long duration for focusing out to make the skip link harder to miss, as described in the WebAIM link */
```

Comment:
Only comment is... might we need to put any of the animation styles behind a `supports()` feature query? Or should the cascade fail gracefully for non supporting browsers? I'm not so familiar with the transition syntax/options.

Reply:
Nevermind - now see your comment at line 13: https://github.com/marko-js/website/pull/81/files/afea158064f265799e625fd5360e1594982027ba#diff-a5b09eacaaac6c1978d4212e68c1e20b0b23fac6cb6381b0c1b30c0ba31e03c1R13